app/css/animator.css

```diff
@@ -505,4 +499,91 @@ a {
 
 ::-webkit-scrollbar-corner       { background-color: #000; }
 
-::-webkit-resizer                { }
\ No newline at end of file
+::-webkit-resizer                { }
+
+/* ======== Preference Windows ======== */
```

Comment:
Will do. There are a few common common styles such as `#sidebar` I've taken from animator.css, I assume it would be better to continue pulling these in from animator.css?